packages/wonder-blocks-dropdown/src/components/__tests__/single-select.test.tsx

```diff
@@ -304,36 +320,52 @@ describe("SingleSelect", () => {
                 },
             );
 
-            // TODO(FEI-5533): Key press events aren't working correctly with
-            // user-event v14. We need to investigate and fix this.
-            it.skip("should select an item when pressing {enter}", async () => {
+            it("should select an item when pressing {enter}", async () => {
                 // Arrange
                 const {userEvent} = doRender(uncontrolledSingleSelect);
                 await userEvent.tab();
                 await userEvent.keyboard("{enter}"); // open
 
+                // Ensure first option is focused, not the opener
+                const firstItem = await screen.findByRole("option", {
+                    name: /item 1/,
+                });
+                expect(firstItem).toHaveFocus();
```

Comment:
In general, it is suggested to have only one expectation in a unit test: https://khanacademy.atlassian.net/wiki/spaces/ENG/pages/98402353/Unit+Testing+Best+Practices#One-Expectation-Per-Test-Case 

I am getting used this too 😅 Could we move this to another test that checks that the first option has focus?

Reply:
Good point, that is something to get used to! My reasoning was that readability/understanding of the next assertion depended on this one. But I can definitely change it!